Integrations/Tenable_io/Tenable_io.py

```diff
@@ -129,7 +129,7 @@ def intersection(lst1, lst2):
 
     return {
         'Type': entryTypes['note'],
-        'Contents': context_obj,
+        'Contents': obj,
```

Comment:
lets return here the object that is not null filtered

Reply:
@Itay4 changed accordingly in the begging of the function.